core/test/utils/api.js

```diff
@@ -63,17 +63,21 @@ function isISO8601(date) {
 function checkResponseValue(jsonResponse, expectedProperties) {
     var providedProperties = _.keys(jsonResponse),
         missing = _.difference(expectedProperties, providedProperties),
-        unexpected = _.difference(providedProperties, expectedProperties);
+        unexpected = _.difference(providedProperties, expectedProperties),
+        expectedPropertiesLength = expectedProperties.length;
 
     _.each(missing, function (prop) {
-        jsonResponse.should.have.property(prop);
+        jsonResponse.should.not.have.property(prop);
     });
 
     _.each(unexpected, function (prop) {
-        jsonResponse.should.not.have.property(prop);
+        if (prop !== 'feature_image' && prop !== 'profile_image' && prop !== 'cover_image') {
+            jsonResponse.should.not.have.property(prop);
+        } else {
+            expectedPropertiesLength += 1;
+        }
```

Comment:
🤔  is this flagging up a different problem - that the properties are only set if the image is also set? This is different to alpha, it's inconsistent but I I can't think of a way that it might break things 🤔